### PR TITLE
Php

### PR DESCRIPTION
**Pull request by istumpf**
Add php example

### CODE REVIEW

File: php/gift_giver/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
```
Comment (matheusazzi) [inline php/gift_giver/.editorconfig]: 👀

Comment (paulodiovani) [reply php/gift_giver/.editorconfig]: php :man_shrugging:

Comment (istumpf) [reply php/gift_giver/.editorconfig]: 🐘

File: examples/php/gift_giver/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Thank you for considering contributing to Lumen! The contribution guide can be found in the [Laravel documentation](https://laravel.com/docs/contributions).
+
+## Security Vulnerabilities
+
+If you discover a security vulnerability within Lumen, please send an e-mail to Taylor Otwell at taylor@laravel.com. All security vulnerabilities will be promptly addressed.
+
+## License
+
+The Lumen framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
```
Comment (paulodiovani) [inline examples/php/gift_giver/README.md]: Need instructions on how to setup and run the project

File: examples/php/gift_giver/app/Http/Controllers/RefreshController.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Attendee;
+
+class RefreshController extends Controller
+{
+    const URL = 'https://api.meetup.com/valetechtalks/events/268854460/rsvps?&sign=true&photo-host=public&fields=answers';
```
Comment (paulodiovani) [inline examples/php/gift_giver/app/Http/Controllers/RefreshController.php]: Would be nice to move to an env var

File: examples/php/gift_giver/app/Http/Controllers/DrawController.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+namespace App\Http\Controllers;
+
+use App\Models\Attendee;
+
+class DrawController extends Controller
+{
+    public function index() {
+        $attendee = Attendee::notAwarded()
+                      ->inRandomOrder()
+                      ->first();
```
Comment (paulodiovani) [inline examples/php/gift_giver/app/Http/Controllers/DrawController.php]: weird indentation :/

**Pull request merged**